django/core/management/utils.py

```diff
@@ -106,3 +106,14 @@ def parse_apps_and_model_labels(labels):
             apps.add(app_config)
 
     return models, apps
+
+
+def preparse_commandline_option(argv, option):
+        parser = CommandParser(add_help=False, allow_abbrev=False)
+        parser.add_argument(option, dest='value')
+
+        try:
+            options, args = parser.parse_known_args(argv[2:])
+            return options.value
```

Comment:
This line can go in "else" of try/except/else since it isn't expected to raise an exception.